src/reconcile/rec_write.c

```diff
@@ -2461,6 +2462,10 @@ __rec_write_wrapup(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page)
     if (F_ISSET(r, WT_REC_HS)) {
         session->reconcile_timeline.hs_wrapup_start = __wt_clock(session);
         ret = __rec_hs_wrapup(session, r);
+        /* Reset the timer if we fail here. */
```

Comment:
We should reset in the err label

Reply:
That sounds sensible, addressed here: [4208c92](https://github.com/wiredtiger/wiredtiger/pull/9055/commits/4208c927f948ae7dce11f2f7de5371f0f452a7d5)